faq/README.md

```diff
@@ -0,0 +1,46 @@
+# FAQ: Frequently Asked Questions 
+
+Django Girls events have been happening all over the world for awhile now, and we get some repeat questions. Hopefully this page will help you if part of the planning process has you confused! 
```

Comment:
I thought the expression was "happening for a while", not "awhile". Am I mistaken?